enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
+Assuming the aforementioned use case and the use of removable media that will be sneakernetted across the gap, the steps for mirroring are:
+
+1. Connect the removable media to a system connected to the internet.
+2. Mirror the images and manifests to a directory on the removable media:
+	```console
+	$ oc adm release mirror --to-dir=<removable-media-path>/mirror quay.io/openshift-release-dev/ocp-release:4.3.0-x86_64
+	```
+3. The enhanced `oc adm release mirror ...` command will output the path, similar to the example below, to a configmap created containing the image signature. Note this path as it will be used later when the configmap is applied to a cluster to be upgraded. 
+	```console
+	Configmap <removable-media-path>/mirror/manifests/sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92 created
+	...
+4. Run any required checks/scrubs on the removable media.
+5. Connect the removable media to a host within the air gapped environment that has access to both the central registry and to any cluster requiring upgrade.
+6. Upload the mirrored images to the central registry:
+	```console
+	$ oc image mirror --from-dir=<removable-media-path>/mirror file://openshift/release:4.3.0* REGISTRY/REPOSITORY
+	```
+7. Use oc to login to a given cluster to be upgraded.
+8. Apply the the mirrored configmap, noted above, to the connected cluster for use by CVO for upgrade image verification:
+	```console
+	$ oc apply -f <removable-media-path>/mirror/manifests/sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
+9. Perform the upgrade:
+	```console
+	$ oc adm upgrade --to-image REGISTRY/REPOSITORY/release@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
 
 ### User Stories
 
 #### Release-signature ConfigMaps
 
-FIXME
+Currently a cluster upgrade can be accomplished on a cluster that does not have an active connection to the internet.
+However manual steps are required to create a ConfigMap containing the signature data required for update image verification.
+This enhancement will automatically create the ConfigMap from the image manifest files.
 
 #### In-cluster Cincinnati
 
 FIXME
 
 ### Risks and Mitigations
 
-FIXME
+One risk considered is that a user familiar with the current behavior of the `oc adm release mirror ...` command may inadvertently apply manifests to thier cluster.
+This has been addressed by requiring the explicit `--apply-manifests` argument to apply manifests to a cluster.
 
 ## Design Details
 
 ### Test Plan
 
-FIXME
+The current e2e test, that makes sure the command always exits successfully and that certain apsects of the content are always present, will be modified to validate the new functioanlity as well.
 
 ### Graduation Criteria
 
-FIXME
-
 ### Upgrade / Downgrade Strategy
 
-FIXME
+The image is included in the payload, but has no content running in a cluster to upgrade.
```

Comment:
I think for `oc` this section is about, as we evolve the CLI API, if/how `oc` callers will update their invocations.  Probably worth pulling in some of [my manifest opacity][1] wording, and shifting up some text that is currently below in the *Version Skew Strategy* section.

[1]: https://github.com/openshift/enhancements/pull/238/commits/25c3888f9317a0846ad737e249792fe3712ac014#r391819245